roles/pulp_webserver/tasks/import_certificates.yml

```
@@ -0,0 +1,18 @@
+---
+- name: Import specified TLS certificate
+  copy:
+    src: "{{ pulp_webserver_ssl_cert }}"
+    dest: "{{ pulp_webserver_tls_folder }}/pulp_webserver.crt"
+    owner: root
+    group: pulp
+    mode: 0600
+  notify: reload {{ pulp_webserver_server }}
+
+- name: Import specified TLS private key
+  copy:
+    src: "{{ pulp_webserver_ssl_key }}"
+    dest: "{{ pulp_webserver_tls_folder }}/pulp_webserver.key"
+    owner: root
+    group: pulp
```

```suggestion
    group: "{{pulp_group }}"
    mode: 0600
  notify: reload {{ pulp_webserver_server }}

- name: Import specified TLS private key
  copy:
    src: "{{ pulp_webserver_ssl_key }}"
    dest: "{{ pulp_webserver_tls_folder }}/pulp_webserver.key"
    owner: root
    group: "{{ pulp_group }}"
```

@dkliban is this invalidating your review?

---

Maybe we should even set the owner and group explicitly on the files when they are generated with the openssl_* modules. (They support that)

---

I can re-review with this change. However, it should continue to work. 

Are we already using the openssl_* ansible modules?

---

What i meant was,...
I'll show you in a suggestion.